### ISSUE TITLE
Make "Animations" demo accessible

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#10831]
* When swiping through elements on the home screen you can select stuff that is currently not on the screen
* VoiceOver needs to read out more information about the selected elements
* When in a detailed screen, you cannot switch to another tab
* more ...

**Comment by goderbauer** [comment]
This has gotten worse. The Animation demo doesn't seem to expose any a11y elements anymore except for the back button :(

**Comment by goderbauer** [comment]
This is better then it used to be, but still needs some work.

**Comment by kf6gpe** [comment]
Closing as obsolete based on @goderbauer's last comment. If we're going to do the work, we should reopen and assign to the frameworks team, please.
